telebot.py

```diff
@@ -1,19 +1,29 @@
+"""This module for work with Telegram BOT API."""
 import requests
 import constants
 import urllib
 import io
 
 
 def tbotxt(text):
+    """Sending text to telegram."""
     params = urllib.parse.quote(text, safe='')  # formatting text for url
-    url_send = "https://api.telegram.org/bot{0}/sendmessage?chat_id={1}&text={2}".format(constants.token, constants.chat_id, params)
+    url_send = (
+        "https://api.telegram.org/bot{0}" +
+        "/sendmessage?chat_id={1}&text={2}").format(
+        constants.token, constants.chat_id, params,
+    )
```

Comment:
It's nice, but I've found much more beautiful way to express it 
```python
url_send = "https://api.telegram.org/bot{}/sendmessage?{}".format(
    constants.token,
    urlencode({
        "chat_id": constants.chat_id,
        "text": params
    })
)
```

*urlencode is inside* `urllib.parser`